api_tests/src/actors/actor.ts

```diff
@@ -277,12 +354,32 @@ export class Actor {
         }
     }
 
+    public async init() {
+        if (!this.swap) {
+            throw new Error("Cannot init nonexistent swap");
+        }
+
+        const response = await this.swap.tryExecuteSirenAction<LedgerAction>(
+            "init",
```

Comment:
I don't think this is a great name. I find it confusing despite being well versed with halight.

Reply:
That is inconsistent with our other action naming though. The name of the action describes the phase and not the actual action that is going to be executed. I see `deploy` being replaced with `init` eventually.